.github/workflows/cdn_release.yaml

```diff
@@ -0,0 +1,51 @@
+name: AWS RUM CDN Release
```

Comment:
nit: Should we name this workflow something that informs us its meant for deprecation (eg - AWS RUM CDN - For Deprecation) ? 
The name right now, might make it seem this is a workflow we run to do a normal release for CDN but the normal release process is the same (using AWS Rum Web Client release workflow)

Reply:
If it's going to be a separate workflow, I agree we should be clear that it is to be used only for rolling back the 'live' CDN distributions.

Reply:
Changed the workflow name to `Rollback CDN Live Version`